src/coreclr/vm/appdomain.cpp

```diff
@@ -51,6 +51,10 @@
 #include "olecontexthelpers.h"
 #endif // FEATURE_COMINTEROP
 
+#if defined(FEATURE_COMWRAPPERS) && !defined(FEATURE_COMINTEROP)
```

Comment:
Let's make this only `FEATURE_COMWRAPPERS` and add duplicate include protection in `rcwrefcache.h` if there is an issue.

Reply:
The duplicate include protection is already in plae in rcwrefcache.h, so just updated appdomain.cpp with only FEATURE_COMWRAPPERS